scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua

```diff
@@ -19,21 +19,11 @@ function onTrade(player,npc,trade)
 end
 
 function onTrigger(player,npc)
-    if player:getVar("MissionStatus") == 4 then
-        local dialog = player:getVar("mandialog")
-
-        if dialog == 0 then
-            player:startEvent(10)
-            player:setVar("mandialog", 1)
-            player:PrintToPlayer("Seems like he wants something")
-        elseif dialog == 1 then
-            player:startEvent(11)
-            player:setVar("mandialog", 2)
-        elseif dialog == 2 then
-            player:startEvent(12)
-            player:setVar("mandialog", 3)
-            player:PrintToPlayer("Seems like he wants some Gobbu Hummus")
-        end
+    if player:getVar("MissionStatus") == 4  or player:getVar("MissionStatus") == 5 then
```

Comment:
Shouldn't that check which mission it is to? I mean lots of missions have a status 4 or status 5 at some point right?